### ISSUE TITLE
Remove node-git dependency to simplify client requirements 

### ISSUE THREAD

**Issue by cgewecke** [codechecks/monorepo#11]
Message is:
```
[nodegit] WARN - Could not finish postinstall
/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /home/travis/.npm/_npx/9492/lib/node_modules/cgewecke-cc-testing/node_modules/nodegit/build/Release/nodegit.node)
```

It's a [known node-git issue](https://www.npmjs.com/package/nodegit#getting-started)  and can be resolved by adding the following to travis.yml:
```yml
addons:
  apt:
    sources:
      - ubuntu-toolchain-r-test
    packages:
      - libstdc++-4.9-dev
```

**Comment by krzkaczor** [comment]
Thanks for raising this issue. We can get rid of using `node-git` by executing git commands directly (which we do anyway). I think it's worth trying — i shall take a closer look tmrw.

**Comment by krzkaczor** [comment]
Okay, turns out that `nodegit` is PITA even on circleci (for example on `circleci/node:latest-browsers` image) so i definately agree that ideally we would delete it. 

The real problem with deleting it is getting patches: https://github.com/codechecks/monorepo/blob/master/packages/client/src/speculativeBranchSelection.ts#L83 It looks like `simple-git` can be an alterative to `nodegit`.

**Comment by krzkaczor** [comment]
Released as `@codechecks/client@0.1.5`

@cgewecke could you give it a try/

**Comment by krzkaczor** [comment]
Btw. it won't happen again with this: https://github.com/codechecks/monorepo/pull/17
